src/mmw/apps/modeling/models.py

```diff
@@ -79,7 +79,7 @@ class Meta:
     name = models.CharField(
         max_length=255)
     project = models.ForeignKey(Project, related_name='scenarios')
-    current_condition = models.BooleanField(
+    is_current_conditions = models.BooleanField(
```

Comment:
The format and type of the modifications and inputs for the different models are so different that running them on existing scenarios would be prohibitive.  For example, modifications to tr55 can be a geometry, where in gwlf-e, they'll likely be declarative "add 5m riparian buffer around all stream networks".  That's why we chose to associate the model package with the project instead.